src/features/tags/components/TagManager/components/TagDialog/index.tsx

```diff
@@ -16,14 +16,16 @@ interface TagDialogProps {
   groups: ZetkinTagGroup[];
   open: boolean;
   onClose: () => void;
-  onDelete: (tagId: number) => void;
+  onDelete?: (tagId: number) => void;
   onSubmit: (tag: NewTag | EditTag) => void;
+  onlyCreateTags?: boolean;
```

Comment:
This feels off to me. Disregarding the (very small) effect that this boolean has on the rendered output, and just going off the name, I'm lead to believe that by default (when this is falsy) this modal does a lot and when the boolean is truthy, it will do _less_.

It feels more natural that by default the modal would do something, and then a property can be supplied to do _more_.

Looking at how the boolean is used, I realize that the thing that the modal _does_ only differs in how the submit action is labelled, and the actual action taken when submitting is handled by the user of the component (in `onSubmit`). Realizing this, I'm thinking the easier solution would be to just let the user of the component supply the submit label, and default it to `undefined`.

```suggestion
  submitLabel?: string;
```